monitor/seaclouds-data-collector/src/main/java/eu/seaclouds/monitor/seacloudsDc/DcService.java

```diff
@@ -0,0 +1,45 @@
+package eu.seaclouds.monitor.seacloudsDc;
+
+import java.util.Properties;
+
+import javax.ws.rs.Produces;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.QueryParam;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Path("/")
+@Produces(MediaType.APPLICATION_JSON)
+public class DcService {
+    static Logger log = LoggerFactory.getLogger(DcService.class);
+    
+    private static final String TYPE_PARAMETER = "type";
+    private static final String ID_PARAMETER = "id";
+    private static final String URL = "url";
+    
+    public DcService(Properties dcProp) {
+        
+        Registry.initialize(dcProp);
+
+    }
+    
+    @POST
+    @Path("/resource")
+    public Response addMonitoredResource(@QueryParam(TYPE_PARAMETER) String type, @QueryParam(ID_PARAMETER) String id,  @QueryParam(URL) String url) {        
```

Comment:
generally in a @POST you want to send a payload not query params. Any particular reason to not use a usual body?